.github/workflows/phpunit.yml

```diff
@@ -0,0 +1,45 @@
+name: PHPUnit tests
+
+on:
+  push:
+    branches: [ master ]
+  schedule:
+    - cron: "0 6 * * 3"
```

Comment:
This is unnecessary in my opinion. Since it doesn't even require any packages it can't really break on it's own.

Reply:
Agree.